app/assets/javascripts/lesson_planner/unit_templates_manager/unit_template_profile/unit_template_profile_assign_button.jsx

```diff
@@ -7,12 +7,14 @@ EC.UnitTemplateProfileAssignButton = React.createClass({
   propsSpecificComponent: function () {
     if (this.props.data.non_authenticated) {
       return <button className='button-green full-width' onClick={this.props.actions.signUp}>Sign Up</button>
-    } else if (!this.props.data.firstAssignButtonClicked) {
+    } else if (!this.props.data.firstAssignButtonClicked && ($(".tab-pane").data().students === true)) {
```

Comment:
@RyanNovas  I think it might be preferrable to have the `areStudentsPresent` value passed in as a prop, so any interaction with the server is localized to the top level component.